src/utils-functions.ts

```diff
@@ -45,3 +45,13 @@ export function getStatusCode(reasonPhrase: string): (number) {
  * @returns {string|undefined} The associated reason phrase (e.g. "Bad Request", "OK")
  * */
 export const getStatusText = getReasonPhrase;
+
+/**
+ * Returns true if the given status code is a valid HTTP status code.
+ *
+ * @param {number} statusCode The HTTP status code
+ * @returns {boolean} True if the status code is valid, false otherwise
+ * */
+export function validateStatusCode(statusCode: number): boolean {
```

Comment:
This could be a type guard and refine the type (`statusCode is StatusCodes` rather than `boolean`).